ext/devcontainer/src/azd/devcontainer-feature.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "azd",
+    "version": "1.0.0",
```

Comment:
```suggestion
    "version": "0.1.0",
```

Reply:
Let's set 0 as the semver major version for this feature when we land it initially, we can move to `1.0.0` after it's had a little bit of bake time and we've validated more of the end to end scenarios.